spec/lib/acts_as_ar_model_spec.rb

```diff
@@ -61,7 +61,7 @@
     end
 
     it "supports where (as an example)" do
-      expect(base_class).to receive(:find).with(:all, :conditions => {:id => 5}).and_return([])
+      expect(base_class).to receive(:find).with(:all, {:conditions => {:id => 5}}).and_return([])
```

Comment:
☝️ These `expect...to receive(...).with` changes are needed because rspec-mocks started supporting ruby 3.0 format for mocking `with` for kwargs.  Once they did that, we're forced to be explicit when we're expecting a hash in the args.  See the rspec-mock commit.

Reply:
Makes sense.  Thanks for the details.